test/unit/unit_test.html

```diff
@@ -51,6 +51,7 @@
   <script src="stream_spec.js"></script>
   <script src="parser_spec.js"></script>
   <script src="api_spec.js"></script>
+  <script src="text_extract.js"></script>
```

Comment:
Perhaps rename this to `text_layer_spec.js` so we can include all text layer related unit tests in there? It's also more consistent with the names of the other test suites.